core/src/components/card/card.stories.tsx

```diff
@@ -50,22 +50,22 @@ export default {
         type: 'text',
       },
     },
-    headerImg: {
-      name: 'Header image',
-      description: 'Toggles an image in the header.',
+    thumbnail: {
+      name: 'Header thumbnail',
+      description: 'Toggles a thumbnail in the header.',
       control: {
         type: 'boolean',
       },
     },
-    headerPlacement: {
-      name: 'Header placement',
-      description: 'Sets the placement of the header, above or below the body image.',
+    imagePlacement: {
+      name: 'Body Image placement',
```

Comment:
Seems like the body image end up above the header and subheader when the option "Below" is selected?

![image](https://github.com/scania-digital-design-system/tegel/assets/48509966/cf21f999-1ff7-4e90-b7ab-0717cac402f9)